src/granite/transactions.cr

```diff
@@ -239,7 +245,7 @@ module Granite::Transactions
   # Returns true if this object hasn't been saved yet.
   @[JSON::Field(ignore: true)]
   @[YAML::Field(ignore: true)]
-  getter? new_record : Bool = true
+  property? new_record : Bool = true
```

Comment:
👍

Reply:
Mmm, I think the new test I added in `update_spec` covers it since it's using the setter, which is the only new thing changing it would do.  I'm not sure what else you could test, especially for just a bool property.